generate_adapter.py

```diff
@@ -40,9 +43,12 @@ def main(
         temperature: A value controlling the randomness of the sampling process. Higher values result in more random
             samples.
     """
+    if not adapter_path:
+        adapter_path = Path("out/adapter/alpaca/lit-stablelm-adapter-finetuned.pth")
+    if not checkpoint_dir:
+        checkpoint_dir = Path(f"checkpoints/stabilityai/stablelm-base-alpha-3b")
```

Comment:
Why are these made optional if we are setting the paths here? This can be the default in the signature

Reply:
It's the same in lit-llama, what's the problem?

Reply:
It's an improvement suggestion. For example, if this was moved to the signature, the help message would show the true default value.

I can propose the same change in lit-llama if you want it to be the same